lib/aspect.js

```diff
@@ -85,7 +85,7 @@ exports.aroundCallback = function(args, hookBefore, hookAfter) {
 		var ret = orig.apply(this, arguments);
 
 		if(hookAfter) {
-			hookAfter(this, arguments);
+			hookAfter(this, arguments, ret);
```

Comment:
@seabaylea Is this change another fix? It doesn't look related to the method name change, unless I'm missing something?